server/server.go

```diff
@@ -455,19 +455,20 @@ func cutPluginCommandF(w http.ResponseWriter, slashCommand *MMSlashCommand, tag,
 		return nil
 	}
 
-	msg := fmt.Sprintf("Tag %s created. Waiting for the artifacts to sign and publish.\nWill report back when the process completes.\nGrab :coffee: and a :doughnut: ", tag)
+	cmdFlags := fmt.Sprintf("tag %s, repo %s, commitSHA %s, force %v", tag, repo, commitSHA, force)
```

Comment:
Lets wrap the flags in the backticks, easier for users to copy paste.

Something like
```
Flags: tag `%s`, repo `%s`, commitSHA `%s`, force `%v`.
```

Also lets include it at the top as opposed to in the sentence. As it'll be easier to incorporate the `username` text after.
e.g

```
Flags: tag `%s`, repo `%s`, commitSHA `%s`, force `%v`.
Tag created. Waiting for the artifacts to sign and publish.\nWill report back when the process completes.\nGrab :coffee: and a :doughnut: 
```